src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,61 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  onAdd: (movie: Movie) => void
+};
+
+export const FindMovie: React.FC<Props> = ({ onAdd }) => {
+  const [query, setQuery] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [movie, setMovie] = useState<Movie | null>(null);
+
+  const getMovieData = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    setIsLoading(true);
+
+    getMovie(query)
```

Comment:
```suggestion
    getMovie(query)
```
Consider making a request with a trimmed query